core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -41,21 +43,39 @@ const (
 
 	// Both VRF and DKG contracts emit this, it's an OCR event.
 	configSetEvent = "ConfigSet"
+
+	// TODO: Add these defaults to the off-chain config, and get better gas estimates
+	// (these current values are very conservative).
+	cacheEvictionWindowSeconds = 60
```

Comment:
If this is in seconds this is actually quite short. 60 seconds is only 5 blocks on Ethereum, and only 30 blocks on Polygon. It _may_ be fine for Ethereum, as 5 block re-orgs are rare, but for Polygon it's a bit too low.

Reply:
Yep this is a TODO